app/helpers/handlers.js

```diff
@@ -411,6 +412,19 @@ const getPage = async (question, request, h) => {
     }
   }
 
+  if (url === 'potential-amount-solar' && getYarValue(request, 'solarPVCost')) {
+    const projectCost = getYarValue(request, 'projectCost')
+    const calculatedGrant = getYarValue(request, 'calculatedGrant')
+    const projectCostSolar = getYarValue(request, 'projectCostSolar')
+    const calculatedGrantSolar = getYarValue(request, 'calculatedGrantSolar')
+
+    const projectCostCombined = projectCost + projectCostSolar
```

Comment:
I used projectCostCombined because in the ticket the following is mentioned:

"You may be able to apply for a grant of up to £A based on the **total 
estimated cost of £B.**"

Where **B** is project cost and solar cost combined.

Reply:
Ahhhhh okay then

Not sure if thats even right but fair enough, will close this comment then :)